src/platform/utilities/api/index.js

```diff
@@ -48,7 +48,7 @@ function isJson(response) {
  */
 export function apiRequest(resource, optionalSettings = {}, success, error) {
   const apiVersion = (optionalSettings && optionalSettings.apiVersion) || 'v0';
-  const baseUrl = `${environment.API_URL}/${apiVersion}`;
+  const baseUrl = `https://staging-api.va.gov/${apiVersion}`;
```

Comment:
Is the env variable suppose to be changed to staging?

Reply:
Nice catch here, need to delete that out! Thanks!

Reply:
Ah ya, this was probably done to use the staging api locally so shouldn't be committed 🙂 Alternatively, you can also do `yarn watch --env.api https://staging-api.va.gov`

Reply:
Whoops I should have refreshed before commenting 😓 🤐